src/components/ContentHighlights/HighlightStepper/HighlightStepperTitleInput.jsx

```diff
@@ -11,37 +11,62 @@ const HighlightStepperTitleInput = () => {
   const highlightTitle = useContextSelector(ContentHighlightsContext, v => v[0].stepperModal.highlightTitle);
   const [titleLength, setTitleLength] = useState(highlightTitle?.length || 0);
   const [isInvalid, setIsInvalid] = useState(false);
-
-  const handleChange = (e) => {
+  /**
+   *  Create seperate useState for FormInput as to differenciate between
+   *  the outer stepper context state and inner input useState.
+   * */
+  const [highlightValue, setHighlightValue] = useState({
+    initialized: false,
+    highlightTitle: highlightTitle || '',
+    titleStepValidationError: undefined,
+    highlightTitleLength: titleLength || 0,
+  });
+  const handleChange = ((e) => {
     if (e.target.value.length > MAX_HIGHLIGHT_TITLE_LENGTH) {
```

Comment:
nit: might recommend creating a variable similar to `const value = e.target.value;` for re-use and to avoid a possibility of needing to use `e.persist()` due to setting state and then trying to access the same `e` variable.